lib/modules/platform/github/index.ts

```diff
@@ -38,6 +38,7 @@ import { regEx } from '../../../util/regex';
 import { sanitize } from '../../../util/sanitize';
 import { fromBase64, looseEquals } from '../../../util/string';
 import { ensureTrailingSlash } from '../../../util/url';
+import { isGithubFineGrainedPersonalAccessToken } from '../../manager/composer/utils';
```

Comment:
Move that function to utils in separate PR before merging this PR

Reply:
That function `isGithubFineGrainedPersonalAccessToken()` along with similar util functions are already in Renovate repo:

https://github.com/renovatebot/renovate/blob/666390468c3969bdcc2941147a0b5b0908110eec/lib/modules/manager/composer/utils.ts#L121-L131

Should I move them to another module, and where is the proper place to put them?

Reply:
https://github.com/renovatebot/renovate/blob/main/lib/util/check-token.ts

I think that is a good place

Reply:
I opened a seperate PR for this change: https://github.com/renovatebot/renovate/pull/23869

Once above PR is merged, I have to rebase this PR to use proper import path. If no further change is required for this PR, I may rebase this PR as soon as possible after https://github.com/renovatebot/renovate/pull/23869 is merged. <- Is it ok to do so?

Reply:
no need to rebase, just merge main and fix conflicts. we'll squash merge anyways

Reply:
ok, I have merged main and updated the import path for `isGithubFineGrainedPersonalAccessToken()`. The PR looks up to date now.